src/client/lib/controllers/user_services.dart

```diff
@@ -16,27 +19,31 @@ class UsersServices{
     });
 
     // Send POST request to /login
-    final response = await http.post(
-      url,
-      headers: {'Content-Type': 'application/JSON'},
-      body: body,
-    );
+    final response = await http
+        .post(
+          url,
+          headers: {'Content-Type': 'application/JSON'},
+          body: body,
+        )
+        .timeout(Duration(seconds: 5));
 
     if (response.statusCode == 200) {
       final data = jsonDecode(response.body);
       final sessionId = response.body;
 
-      // Store the session ID in shared preferences
+      // Store the session ID temporarily using shared preferences library in the future will move to user model
       final prefs = await SharedPreferences.getInstance();
-      await prefs.setString('x-auth-token', sessionId);
+      await prefs.setString('username', username);
+      await prefs.setString('password', password);
+      await prefs.setString('sessionId', sessionId);
```

Comment:
this was correct before, the result of the `sign_up` call be stored into `x-auth-token` (mainly to follow RFC7235)

Reply:
also the code still references `x-auth-token`, so reverting this change is all that needs to be done